recipes/metabolishmm/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "metabolishmm" %}
+{% set version = "1.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/elizabethmcd/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: c9c07059fadcfb4597804207def02a41bfff67262ca58080d7f49baabe700bf4
+
+build:
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - biopython
+    - prodigal >=2.6.2
+    - hmmer ==3.2.1
+    - fastani >=1.2
+    - fasttree >=2.1.9
+    - matplotlib-base
+    - pandas
+    - seaborn
+    - numpy
+    - scipy
+    - r-tidyverse
+    - r-reshape2
```

Comment:
I see this in the tests:
```
+ single-marker-phylogeny -h
You do not have MAFFT installed in your path. Please fix this.
```
So you should
* change the test so that it will fail if this message is thrown (use `grep` to search for expected output)
* add `mafft` to the run dependencies

It would be best to do this in two separate commits so that you can verify that the test fails as expected

Reply:
@marcelm thank you again for your review.

The second part of adding mafft to the dependencies is clear to me. Sorry for missing that. But I'm not sure how to change the test behaviour.

I thought about something like: `single-marker-phylogeny -h | grep mafft`, but how can I indicate "if there is output to this command, fail the test" ?

Thank you for any assistance you can provide.

Reply:
Yes, what you sugest in your edit is what I meant: Just grep for some text that would normally be in the output of the tool!